doc/changelog/08-tools/11523-coqdep+refactor2.rst

```diff
@@ -1,7 +1,10 @@
 - **Changed:**
-  Internal options and behavior of ``coqdep`` have changed, in particular
-  options ``-w``, ``-D``, ``-mldep``, and ``-dumpbox`` have been removed,
-  and ``-boot`` will not load any path by default, ``-R/-Q`` should be
-  used instead
-  (`#11523 <https://github.com/coq/coq/pull/11523>`_,
-  by Emilio Jesus Gallego Arias).
+  Internal options and behavior of ``coqdep`` have changed. ``coqdep``
+  no longer works as a replacement for ``ocamldep``, thus ``.ml``
+  files are not supported as input. Also, several deprecated options
+  have been removed: ``-w``, ``-D``, ``-mldep``, ``-prefix``,
+  ``-slash``, and ``-dumpbox``. Passing ``-boot`` to ``coqdep`` will
+  not load any path by default now, ``-R/-Q`` should be used instead.
+  (`#11523 <https://github.com/coq/coq/pull/11523>`_ and
+  `#11589 <https://github.com/coq/coq/pull/11589>`_,
+   by Emilio Jesus Gallego Arias).
```

Comment:
I think sphinx didn't like that this line starts with 3 spaces (not sure if that's why it fails but worth a try)

Reply:
Yes, that's the problem.